web/templates/settings/generate_integration_url_modal.hbs

```diff
@@ -0,0 +1,33 @@
+<div class="new-style">
+    <div class="input-group">
+        <label for="integration-input">{{t "Integration"}}</label>
+        <input type="text" id="integration-input" class="modal_text_input integration-url-parameter" autocomplete="off"/>
+    </div>
+    <div class="input-group">
+        <div class="integration-url-stream-wrapper integration-url-parameter">
+            {{> ../dropdown_widget_with_label
+              widget_name="integration-url-stream"
+              label=(t "Stream")}}
+        </div>
+        <p id="stream-not-specified-notice">{{t "If you do not specify a stream, the bot will send notifications via PMs to the creator of the bot."}}</p>
```

Comment:
Having this block of text inline feels awkward. I feel like ideally we'd just have the default choice in this stream picker be "Direct message to me" and adjust the label a bit so that feels natural, e.g. "Where to send notifications"

Reply:
Yes, that sounds right. This should also clear things up in the form itself.